pkgs/development/libraries/fflas-ffpack/default.nix

```diff
@@ -6,20 +6,29 @@ stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "fflas-ffpack";
   version = "2.3.2";
+
   src = fetchFromGitHub {
     owner = "linbox-team";
     repo = "${pname}";
     rev = "v${version}";
     sha256 = "1cqhassj2dny3gx0iywvmnpq8ca0d6m82xl5rz4mb8gaxr2kwddl";
   };
+
   checkInputs = [
     gmpxx
   ];
+
   nativeBuildInputs = [
     autoreconfHook
     pkgconfig
-  ] ++ stdenv.lib.optionals doCheck checkInputs;
+  ] ++ stdenv.lib.optionals doInstallCheck checkInputs;
+
   buildInputs = [ givaro openblas];
+
+  # doInstallCheck needed for darwin
+  doInstallCheck = true;
+  installCheckTarget = "check";
```

Comment:
Same as the other tickets, please expand and report upstream. Does the same issue actually affect all 3 packages? I could understand linbox and givaro since they are from the same people, but this seems weird.

Reply:
No this was my mistake, I saw a segfault and assumed it was the same problem, but it was actually openblas. But then it wasn't running the tests so it was looking like it was working.